charts/ziti-controller/values.yaml

```diff
@@ -156,6 +162,8 @@ dbFile:         ctrl.db
 ctrlPlaneCaDir: ctrl-plane-cas
 # -- filename of the ctrl plane trust bundle
 ctrlPlaneCasFile: ctrl-plane-cas.crt
+# -- file path of the spire socket mount
+spireSocketMnt: /run/spire/sockets
```

Comment:
@kevinlmadison Shouldn't the socket path be defined as part of the `spireAgent` dictionary? I know this top level is cluttered with various filenames and paths. I'd prefer to keep the spire stuff together rather than prioritize file and path values being organized together.

Reply:
absolutely!

Reply:
Committed those suggestions.